.cbmc-batch/jobs/aws_cryptosdk_sig_get_privkey/aws_cryptosdk_sig_get_privkey_harness.c

```diff
@@ -23,28 +23,27 @@
 #include <cipher_openssl.h>
 
 void aws_cryptosdk_sig_get_privkey_harness() {
-    /* arguments */
-    struct aws_cryptosdk_sig_ctx *ctx = can_fail_malloc(sizeof(struct aws_cryptosdk_sig_ctx));
+    /* Nondet Input */
+    struct aws_cryptosdk_sig_ctx *ctx = ensure_nondet_sig_ctx_has_allocated_members();
     struct aws_allocator *alloc       = can_fail_allocator();
-    struct aws_string *priv_key;
+    struct aws_string *priv_key       = ensure_string_is_allocated_nondet_length();
```

Comment:
This is OK for this PR, but we should really rename the `ensure` functions to make it clear that they may or may not allocate the fields.

Reply:
I've made github issues for those both in ESDK (https://github.com/aws/aws-encryption-sdk-c/issues/614) and c-common (https://github.com/awslabs/aws-c-common/issues/723)